coordinator/src/main/scala/filodb/coordinator/ProtoConverters.scala

```diff
@@ -0,0 +1,376 @@
+package filodb.coordinator
+
+import filodb.core.query.Filter.Equals
+import filodb.core.query._
+import filodb.core.store.{AllChunkScan, InMemoryChunkScan, TimeRangeChunkScan, WriteBufferChunkScan}
+import filodb.grpc.ExecPlans
+import filodb.grpc.ExecPlans.ExecPlan.DispatcherCase
+import filodb.grpc.ExecPlans.ExecPlanContainer.ExecPlanCase
+import filodb.query.QueryCommand
+import filodb.query.exec.{LeafExecPlan, LocalPartitionDistConcatExec, MultiSchemaPartitionsExec}
+
+import java.util.concurrent.TimeUnit
+import scala.collection.JavaConverters
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+// scalastyle:off number.of.methods
+// scalastyle:off number.of.types
+// scalastyle:off file.size.limit
+// scalastyle:off method.length
+object ProtoConverters {
```

Comment:
Why not use the existing ``ProtoConverters``, especially when the ExecPlans are in query? Please use the same  object and add your converters there. Also make sure you tests are written for each and every converter.

Reply:
dispatchers are in the coordinator, otherwise, I would have kept them in the original ProtoConverters, in fact I started writing the logic in the original ProtoConverters.